### PR TITLE
[SMALLFIX] Some code refactoring

### PR DESCRIPTION
**Pull request by peisun1115**

### CODE REVIEW

**Comment by ifcharming** [comment]
Jenkins, test this please

**Comment by peisun1115** [comment]
@ifcharming  Not sure why it is failing. I am taking a look.

**Comment by peisun1115** [comment]
Jenkins, test this please

**Comment by peisun1115** [comment]
Jenkins, test this please

**Comment by peisun1115** [comment]
@ifcharming The test passed locally. I will make sure it passes on jenkins. Can you take a look at this PR?

**Comment by ifcharming** [comment]
LGTM, might as well have @calvinjia take a look.

**Comment by peisun1115** [comment]
@calvinjia Can you help to take a look?

File: core/client/src/main/java/alluxio/client/block/AlluxioBlockStore.java [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 import alluxio.Constants;
 import alluxio.client.ClientContext;
+import alluxio.client.file.options.InStreamOptions;
```
Comment (calvinjia) [inline core/client/src/main/java/alluxio/client/block/AlluxioBlockStore.java]: Does it make sense to reuse `file` options at a `block` level?

File: core/common/src/thrift/block_worker.thrift [hunk 2]
```diff
@@ -63,34 +75,42 @@ service BlockWorkerClientService extends common.AlluxioService {
...
    * Used to unlock a block after the block is accessed, if the block is to be removed, delete the
    * block file. return true if successfully unlock the block, return false if the block is not
    * found or failed to delete the block.
    */
-  bool unlockBlock( /** the id of the block being accessed */ 1: i64 blockId,
-      /** the id of the current session */ 2: i64 sessionId)
+  bool unlockBlock(
+    /** the id of the block being accessed */ 1: i64 blockId,
+    /** the id of the current session */ 2: i64 sessionId,
+    )
```
Comment (calvinjia) [inline core/common/src/thrift/block_worker.thrift]: Why the change in format?

Comment (peisun1115) [inline core/common/src/thrift/block_worker.thrift]: done for all

File: core/client/src/main/java/alluxio/client/block/AlluxioBlockStore.java (hunk 1 above)
Comment (peisun1115) [inline core/client/src/main/java/alluxio/client/block/AlluxioBlockStore.java]: ack

**Comment by peisun1115** [comment]
@calvinjia PTAL? thanks

File: core/client/src/main/java/alluxio/client/block/AlluxioBlockStore.java (hunk 1 above)
Comment (calvinjia) [inline core/client/src/main/java/alluxio/client/block/AlluxioBlockStore.java]: Hmm okay, it seems like we already using it this way, I've opened a ticket to improve this: https://alluxio.atlassian.net/browse/ALLUXIO-2423
